test/config_test.exs

```diff
@@ -53,4 +53,16 @@ defmodule ConfigTest do
 
     assert NewRelic.Config.feature?(:error_collector)
   end
+
+  test "Can configure httpoison options" do
+    Application.put_env(:new_relic_agent, :httpoison_opts,
+      ssl: [{:versions, [:"tlsv1.2"]}],
```

Comment:
Just FYI, this is actually a dangerous example. The explicit`ssl` options override the Hackney defaults, which means peer certificate verification is not active. See below, where the first request fails (as it should) but the second one succeeds:

```elixir
iex(1)> HTTPoison.get("https://expired.badssl.com/", [], [])
{:error, %HTTPoison.Error{id: nil, reason: {:tls_alert, 'certificate expired'}}}

iex(2)> HTTPoison.get("https://expired.badssl.com/", [], ssl: [versions: [:"tlsv1.2"]])        
{:ok,
 %HTTPoison.Response{
   body: "<!DOCTYPE html>\n<html>\n<head>\n" <> ...
```